pulsar_devkit/simple_examples/HLSIPs/simple_algo_array_ref.cpp

```diff
@@ -0,0 +1,17 @@
+#include <cmath>
+#include <algorithm>
+#include "ap_int.h"
+#include "src/simple_algo_array.h"
+
+void simple_algo_array_ref( ap_int<32> inA[N_INPUTS], ap_int<32> inB[N_INPUTS], ap_int<32> outA[N_OUTPUTS] ) {
+  //#pragma HLS RESOURCE variable=inA core=RAM_1P_BRAM
+  //#pragma HLS RESOURCE variable=inB core=RAM_1P_BRAM
+#pragma HLS INTERFACE bram   port=inA
```

Comment:
@ssevova I think don't think you want the pragmas in the reference algorithm

Reply:
whoops sorry, will remove